README.md

```diff
@@ -1,2 +1,48 @@
 Towtruck is an application to aid maintenence of dxw's repos. 
 It aims to make it easier to keep on top of which repos need updates applying.
+
+
+## Configuration
+
+Towtruck is set up as a [GitHub App](https://docs.github.com/en/apps).
+
+
+### GitHub App settings
+
+The first step is to register a new app as described [here](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app):
```

Comment:
I think we should set up the app under the dxw org and share the env vars needed in 1Password rather than require each user to create their own GH app

Reply:
I agree that we should do that as part of the deployment to production, but in the spirit of open source it's probably a good idea to provide instructions for others if they think it would be useful.

Reply:
that is a good point I hadn't considered. I think it would be good to have both but we can add the dxw-specific setup in a subsequent PR